### ISSUE TITLE
Animated Gif - division by zero

### ISSUE THREAD

**Issue by AngeloCresta** [dotnet/core#6482]
### Description
Binding an animated gif to a picture box will cause a division by zero error

### configuration
VS 2022 P2,. Net Core 6 preview 6 on Windows 10 21H1 (x64) 

### Regression?
Worked on. Net 5

### Other information - Stack Trace

CoreCLR Version: 6.0.21.35212
.NET Version: 6.0.0-preview.6.21352.12
Description: The process was terminated due to an unhandled exception.
Exception Info: System.DivideByZeroException: Attempted to divide by zero.
   at System.Math.DivRem(Int64 a, Int64 b, Int64& result) in System.Private.CoreLib.dll:token 0x600046a+0x0
   at System.Drawing.ImageAnimator.ImageInfo.AdvanceAnimationBy(Int64 milliseconds) in System.Drawing.Common.dll:token 0x6000bed+0x31
   at System.Drawing.ImageAnimator.AnimateImages() in System.Drawing.Common.dll:token 0x60002e0+0x3d
   at System.Threading.Thread.StartHelper.Callback(Object state) in System.Private.CoreLib.dll:token 0x600288d+0x0
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) in System.Private.CoreLib.dll:token 0x6002a2d+0x40
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) in System.Private.CoreLib.dll:token 0x6002a2d+0x80
   at System.Threading.Thread.StartCallback() in System.Private.CoreLib.dll:token 0x600280f+0xe

**Comment by wfurt** [comment]
should we transfer this to runtime @safern?

**Comment by safern** [comment]
Yes. I'll transfer it and since it is a regression I will mark it as 6.0.0
